src/DIRAC/WorkloadManagementSystem/Service/WMSAdministratorHandler.py

```diff
@@ -32,136 +32,106 @@ def initializeHandler(cls, svcInfoDict):
 
         return S_OK()
 
-    types_setSiteMask = [list]
-
+    @deprecated("no-op RPC")
     def export_setSiteMask(self, siteList):
         """Set the site mask for matching. The mask is given in a form of Classad string.
-
         :param list siteList: site, status
-
         :return: S_OK()/S_ERROR()
         """
-        credDict = self.getRemoteCredentials()
-        maskList = [(site, "Active") for site in siteList]
-        return self.jobDB.setSiteMask(maskList, credDict["username"], "No comment")
+        return S_OK()
 
     ##############################################################################
     types_getSiteMask = []
 
     @classmethod
+    @deprecated("no-op RPC")
     def export_getSiteMask(cls, siteState="Active"):
         """Get the site mask
-
```

Comment:
need a new line before the `:param` blocks.
this causes the errors in read the docs builds